examples/03_second_level_models/plot_second_level_association_test.py

```diff
@@ -68,11 +68,55 @@
 _, threshold = map_threshold(z_map, alpha=.05, height_control='fdr')
 
 ###########################################################################
-#Let us plot the second level contrast at the computed thresholds 
+# Let us plot the second level contrast at the computed thresholds
 from nilearn import plotting
 plotting.plot_stat_map(
     z_map, threshold=threshold, colorbar=True,
     title='Group-level association between motor activity \n'
     'and reading fluency (fdr<0.05')
 
 plotting.show()
+
+##########################################################################
+# Computing the (corrected) p-values with parametric test to compare with
+# non parametric test
+from nilearn.image import math_img
+from nilearn.input_data import NiftiMasker
+p_val = model.compute_contrast('fluency', output_type='p_value')
+masker = NiftiMasker(mask_strategy='background').fit(p_val)
```

Comment:
Also, this looks spurious.

Reply:
Yes, using `z_map` instead of `p_val` looks nicer. Okay for you ?

Reply:
No, I think we should  not create a masker here.